benchmarks/src/main/scala/monix/connect/benchmarks/parquet/ParquetWriterBenchmark.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2020-2020 by The Monix Connect Project Developers.
+ * See the project homepage at: https://connect.monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.connect.benchmarks.parquet
+
+import monix.connect.parquet.{Parquet, ParquetSink}
+import monix.eval.{Coeval, Task}
+import monix.execution.Scheduler
+import monix.reactive.Observable
+import org.apache.avro.generic.GenericRecord
+import org.apache.parquet.hadoop.ParquetWriter
+import org.openjdk.jmh.annotations.{
+  BenchmarkMode,
+  Fork,
+  Measurement,
+  Mode,
+  OutputTimeUnit,
+  Scope,
+  State,
+  Threads,
+  Warmup,
+  _
+}
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, Future}
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.Throughput))
+@Measurement(iterations = 4)
+@Warmup(iterations = 1)
+@Fork(1)
+@Threads(5)
+class ParquetWriterBenchmark extends ParquetBenchFixture {
+
+  var size: Int = 10
+  val s = Scheduler.io("parquet-writer-benchmark")
+
+  @Benchmark
+  def unsafe(): Unit = {
+    val file: String = genFilePath.value()
+    val records: List[GenericRecord] = genPersons(size).sample.get.map(personToRecord)
```

Comment:
All benchmarks should compare vs exactly the same data to give us any insight

I would recommend to do a setup like here: https://github.com/monix/monix/blob/series/3.x/benchmarks/shared/src/main/scala/monix/benchmarks/ChunkedEvalFilterMapSumBenchmark.scala#L83

Basically put records to a var outside the methods, fill it in `setup` and then use in benchmarks